### PR TITLE
Add compat data for Headers (Fetch API)

### PR DESCRIPTION
**Pull request by maboa**
This one is a little odd as it contains an entry for both "`getAll()` removed" (see the table here 
https://developer.mozilla.org/en-US/docs/Web/API/Headers)  and `getAll()`. I'm pretty sure the former is redundant - but am leaving it in just in case.

### CODE REVIEW

File: api/Headers.json [hunk 1]
```diff
@@ -0,0 +1,908 @@
...
+        "support": {
+          "webview_android": {
+            "version_added": "40"
+          },
+          "chrome": {
+            "version_added": "52"
+          },
+          "chrome_android": {
+            "version_added": "40"
+          },
```
Comment (jpmedley) [inline api/Headers.json]: These should all be 42.

File: api/Headers.json [hunk 2]
```diff
@@ -0,0 +1,908 @@
...
+          "description": "Lexicographical sorting, and values from duplicate header names combined when iterated.",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": "42"
+            },
+            "chrome_android": {
+              "version_added": "44"
```
Comment (jpmedley) [inline api/Headers.json]: Where did you get the information about lexicographical sorting?

File: api/Headers.json [hunk 3]
```diff
@@ -0,0 +1,908 @@
...
+              },
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
+                  }
+                ]
+              }
```
Comment (jpmedley) [inline api/Headers.json]: Whoever added this to the old compat table didn't get everything. These version added numbers apply to Android and Webview as well as desktop.

File: api/Headers.json [hunk 4]
```diff
@@ -0,0 +1,908 @@
...
+              {
+                "version_added": "28",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
+                  }
+                ]
+              }
+            ],
```
Comment (jpmedley) [inline api/Headers.json]: This data applies to Opera Android as well.

File: api/Headers.json [hunk 5]
```diff
@@ -0,0 +1,908 @@
...
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
+                  }
+                ]
+              }
+            ],
```
Comment (jpmedley) [inline api/Headers.json]: These numbers also apply to Android and Webview.

File: api/Headers.json [hunk 6]
```diff
@@ -0,0 +1,908 @@
...
+              {
+                "version_added": "28",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
+                  }
+                ]
+              }
+            ],
```
Comment (jpmedley) [inline api/Headers.json]: These numbers also apply to Opera Android.

File: api/Headers.json [hunk 7]
```diff
@@ -0,0 +1,908 @@
...
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
+                  }
+                ]
+              }
+            ],
```
Comment (jpmedley) [inline api/Headers.json]: Copy numbers to Android and Webview.

File: api/Headers.json [hunk 8]
```diff
@@ -0,0 +1,908 @@
...
+              {
+                "version_added": "28",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
+                  }
+                ]
+              }
+            ],
```
Comment (jpmedley) [inline api/Headers.json]: Copy to Opera Android.

File: api/Headers.json [hunk 9]
```diff
@@ -0,0 +1,908 @@
...
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
+                  }
+                ]
+              }
+            ],
```
Comment (jpmedley) [inline api/Headers.json]: Copy to Android and Webview

File: api/Headers.json [hunk 10]
```diff
@@ -0,0 +1,908 @@
...
+              {
+                "version_added": "28",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
+                  }
+                ]
+              }
+            ],
```
Comment (jpmedley) [inline api/Headers.json]: Copy to Opera Android.

File: api/Headers.json [hunk 11]
```diff
@@ -0,0 +1,908 @@
...
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
+                  }
+                ]
+              }
+            ],
```
Comment (jpmedley) [inline api/Headers.json]: Copy to Android and Webview.

File: api/Headers.json [hunk 12]
```diff
@@ -0,0 +1,908 @@
...
+              {
+                "version_added": "28",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
+                  }
+                ]
+              }
+            ],
```
Comment (jpmedley) [inline api/Headers.json]: Copy to Opera Android.

File: api/Headers.json [hunk 13]
```diff
@@ -0,0 +1,908 @@
...
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
+                  }
+                ]
+              }
+            ],
```
Comment (jpmedley) [inline api/Headers.json]: Copy to Android and Webview.

File: api/Headers.json [hunk 14]
```diff
@@ -0,0 +1,908 @@
...
+              {
+                "version_added": "28",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
+                  }
+                ]
+              }
+            ],
```
Comment (jpmedley) [inline api/Headers.json]: Copy to Opera Android.

File: api/Headers.json (hunk 1 above)
Comment (maboa) [reply api/Headers.json]: :+1:

File: api/Headers.json (hunk 2 above)
Comment (maboa) [reply api/Headers.json]: https://developer.mozilla.org/en-US/docs/Web/API/Headers Here but I made an error - they should be null according to the old compat table.

File: api/Headers.json (hunk 3 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 4 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 5 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 6 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 7 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 8 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 9 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 10 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 11 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 12 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 13 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 14 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json [hunk 15]
```diff
@@ -0,0 +1,1106 @@
...
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": [
+            {
+              "version_added": "52",
```
Comment (Elchi3) [inline api/Headers.json]: Remove the entry with 52 and the note. This belongs to the `get()` entry and not here.

File: api/Headers.json [hunk 16]
```diff
@@ -0,0 +1,1106 @@
...
+            },
+            {
+              "version_added": "39"
+            },
+            {
+              "version_added": "34",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: dom.fetch.enabled

File: api/Headers.json [hunk 17]
```diff
@@ -0,0 +1,1106 @@
...
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "advanced_header_methods": {
```
Comment (Elchi3) [inline api/Headers.json]: Remove the whole "advanced_header_methods entry. It is redundant with the `entries`, `keys`, `values` data.

File: api/Headers.json [hunk 18]
```diff
@@ -0,0 +1,1106 @@
...
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            }
+          }
+        }
+      },
+      "getall_removed": {
```
Comment (Elchi3) [inline api/Headers.json]: Remove the whole entry for "getall_removed". This is redundant with the info in `getAll`.

File: api/Headers.json [hunk 19]
```diff
@@ -0,0 +1,1106 @@
+{
+  "api": {
+    "Headers": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Headers",
+        "support": {
+          "webview_android": {
+            "version_added": "42"
```
Comment (Elchi3) [inline api/Headers.json]: I think this is the complete thing for the Chrome family.

```json
"webview_android": [
              {
                "version_added": "42"
              },
              {
                "version_added": "41",
                "flags": [
                  {
                    "type": "preference",
                    "name": "Experimental Web Platform Features"
                  }
                ]
              }
            ],
            "chrome": [
              {
                "version_added": "42"
              },
              {
                "version_added": "41",
                "flags": [
                  {
                    "type": "preference",
                    "name": "Experimental Web Platform Features"
                  }
                ]
              }
            ],
            "chrome_android": [
              {
                "version_added": "42"
              },
              {
                "version_added": "41",
                "flags": [
                  {
                    "type": "preference",
                    "name": "Experimental Web Platform Features"
                  }
                ]
              }
            ],
```

File: api/Headers.json [hunk 20]
```diff
@@ -0,0 +1,1106 @@
...
+            "webview_android": [
+              {
+                "version_added": "42"
+              },
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 21]
```diff
@@ -0,0 +1,1106 @@
...
+            "chrome": [
+              {
+                "version_added": "42"
+              },
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 22]
```diff
@@ -0,0 +1,1106 @@
...
+            "chrome_android": [
+              {
+                "version_added": "42"
+              },
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 23]
```diff
@@ -0,0 +1,1106 @@
...
+            "firefox": [
+              {
+                "version_added": "39"
+              },
+              {
+                "version_added": "34",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: dom.fetch.enabled

File: api/Headers.json [hunk 24]
```diff
@@ -0,0 +1,1106 @@
...
+            "opera": [
+              {
+                "version_added": "29"
+              },
+              {
+                "version_added": "28",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 25]
```diff
@@ -0,0 +1,1106 @@
...
+            "opera_android": [
+              {
+                "version_added": "29"
+              },
+              {
+                "version_added": "28",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 26]
```diff
@@ -0,0 +1,1106 @@
...
+            "webview_android": [
+              {
+                "version_added": "42"
+              },
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 27]
```diff
@@ -0,0 +1,1106 @@
...
+            "chrome": [
+              {
+                "version_added": "42"
+              },
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 28]
```diff
@@ -0,0 +1,1106 @@
...
+            "chrome_android": [
+              {
+                "version_added": "42"
+              },
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 29]
```diff
@@ -0,0 +1,1106 @@
...
+            "firefox": [
+              {
+                "version_added": "39"
+              },
+              {
+                "version_added": "34",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: dom.fetch.enabled

File: api/Headers.json [hunk 30]
```diff
@@ -0,0 +1,1106 @@
...
+            "opera": [
+              {
+                "version_added": "29"
+              },
+              {
+                "version_added": "28",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 31]
```diff
@@ -0,0 +1,1106 @@
...
+            "opera_android": [
+              {
+                "version_added": "29"
+              },
+              {
+                "version_added": "28",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 32]
```diff
@@ -0,0 +1,1106 @@
...
+            "webview_android": [
+              {
+                "version_added": "42"
+              },
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 33]
```diff
@@ -0,0 +1,1106 @@
...
+            "chrome": [
+              {
+                "version_added": "42"
+              },
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 34]
```diff
@@ -0,0 +1,1106 @@
...
+            "chrome_android": [
+              {
+                "version_added": "42"
+              },
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 35]
```diff
@@ -0,0 +1,1106 @@
...
+            "firefox": [
+              {
+                "version_added": "39"
+              },
+              {
+                "version_added": "34",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: dom.fetch.enabled

File: api/Headers.json [hunk 36]
```diff
@@ -0,0 +1,1106 @@
...
+            "opera": [
+              {
+                "version_added": "29"
+              },
+              {
+                "version_added": "28",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 37]
```diff
@@ -0,0 +1,1106 @@
...
+            "opera_android": [
+              {
+                "version_added": "29"
+              },
+              {
+                "version_added": "28",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 38]
```diff
@@ -0,0 +1,1106 @@
...
+            "webview_android": [
+              {
+                "version_added": "42"
+              },
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 39]
```diff
@@ -0,0 +1,1106 @@
...
+            "chrome": [
+              {
+                "version_added": "42"
+              },
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 40]
```diff
@@ -0,0 +1,1106 @@
...
+            "chrome_android": [
+              {
+                "version_added": "42"
+              },
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 41]
```diff
@@ -0,0 +1,1106 @@
...
+              },
+              {
+                "version_added": "39"
+              },
+              {
+                "version_added": "34",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: dom.fetch.enabled

File: api/Headers.json [hunk 42]
```diff
@@ -0,0 +1,1106 @@
...
+            "opera": [
+              {
+                "version_added": "29"
+              },
+              {
+                "version_added": "28",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 43]
```diff
@@ -0,0 +1,1106 @@
...
+            "opera_android": [
+              {
+                "version_added": "29"
+              },
+              {
+                "version_added": "28",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 44]
```diff
@@ -0,0 +1,1106 @@
...
+            "firefox": [
+              {
+                "version_added": "39"
+              },
+              {
+                "version_added": "34",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: dom.fetch.enabled

File: api/Headers.json [hunk 45]
```diff
@@ -0,0 +1,1106 @@
...
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": [
+              {
+                "version_added": "39"
```
Comment (Elchi3) [inline api/Headers.json]: Add `"version_removed": "52"`

File: api/Headers.json [hunk 46]
```diff
@@ -0,0 +1,1106 @@
...
+            "webview_android": [
+              {
+                "version_added": "42"
+              },
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 47]
```diff
@@ -0,0 +1,1106 @@
...
+            "chrome": [
+              {
+                "version_added": "42"
+              },
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 48]
```diff
@@ -0,0 +1,1106 @@
...
+            "chrome_android": [
+              {
+                "version_added": "42"
+              },
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 49]
```diff
@@ -0,0 +1,1106 @@
...
+            "firefox": [
+              {
+                "version_added": "39"
+              },
+              {
+                "version_added": "34",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: dom.fetch.enabled

File: api/Headers.json [hunk 50]
```diff
@@ -0,0 +1,1106 @@
...
+            "opera": [
+              {
+                "version_added": "29"
+              },
+              {
+                "version_added": "28",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 51]
```diff
@@ -0,0 +1,1106 @@
...
+            "opera_android": [
+              {
+                "version_added": "29"
+              },
+              {
+                "version_added": "28",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 52]
```diff
@@ -0,0 +1,1106 @@
...
+            "webview_android": [
+              {
+                "version_added": "42"
+              },
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 53]
```diff
@@ -0,0 +1,1106 @@
...
+            "chrome": [
+              {
+                "version_added": "42"
+              },
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 54]
```diff
@@ -0,0 +1,1106 @@
...
+            "chrome_android": [
+              {
+                "version_added": "42"
+              },
+              {
+                "version_added": "41",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 55]
```diff
@@ -0,0 +1,1106 @@
...
+            "firefox": [
+              {
+                "version_added": "39"
+              },
+              {
+                "version_added": "34",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: dom.fetch.enabled

File: api/Headers.json [hunk 56]
```diff
@@ -0,0 +1,1106 @@
...
+            "opera": [
+              {
+                "version_added": "29"
+              },
+              {
+                "version_added": "28",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 57]
```diff
@@ -0,0 +1,1106 @@
...
+            "opera_android": [
+              {
+                "version_added": "29"
+              },
+              {
+                "version_added": "28",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": ""
```
Comment (Elchi3) [inline api/Headers.json]: "Experimental Web Platform Features"

File: api/Headers.json [hunk 58]
```diff
@@ -0,0 +1,1106 @@
...
+            }
+          ],
+          "firefox_android": {
+            "version_added": "44"
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": "29"
```
Comment (Elchi3) [inline api/Headers.json]:
```json
            "opera": [
              {
                "version_added": "29"
              },
              {
                "version_added": "28",
                "flags": [
                  {
                    "type": "preference",
                    "name": "Experimental Web Platform Features"
                  }
                ]
              }
            ],
            "opera_android": [
              {
                "version_added": "29"
              },
              {
                "version_added": "28",
                "flags": [
                  {
                    "type": "preference",
                    "name": "Experimental Web Platform Features"
                  }
                ]
              }
            ],
```

File: api/Headers.json (hunk 19 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 15 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 16 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 58 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 17 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 18 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 20 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 21 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 22 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 23 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 24 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 25 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 26 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 27 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 28 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 29 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 30 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 31 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 32 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 33 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 34 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 35 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 36 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 37 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 38 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 39 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 40 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 41 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 42 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 43 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 44 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 46 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 47 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 48 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 49 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 50 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 51 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 52 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 53 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 54 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 55 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 56 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 57 above)
Comment (maboa) [reply api/Headers.json]: 👍

File: api/Headers.json (hunk 45 above)
Comment (maboa) [reply api/Headers.json]: 👍

**Pull request merged**